main.go

```diff
@@ -88,6 +88,7 @@ func Router(server *socketio.Server) *mux.Router {
 	r.HandleFunc("/organizations/{id}/url", au.IsAuthenticated(orgs.UpdateURL)).Methods("PATCH")
 	r.HandleFunc("/organizations/{id}/name", au.IsAuthenticated(orgs.UpdateName)).Methods("PATCH")
 	r.HandleFunc("/organizations/{id}/logo", au.IsAuthenticated(orgs.UpdateLogo)).Methods("PATCH")
+	r.HandleFunc("/organizations/{id}/members/{mem_id}/custom-emoji", au.IsAuthenticated(orgs.CustomEmoji)).Methods("POST")
```

Comment:
This happens on organization resource, not member, so member id should come from the body request. Batter still you can extract it from the logged in user. 

"CustomEmoji" doesn't really tell what your function is about at a glance. Are you adding , updating or deleting?
And I think a patch request is appropriate not post